app/frontend/js/TestDestination.js

```diff
@@ -1,6 +1,6 @@
 import { bindTestForm } from './utils/test-form';
 
-bindTestForm('test', 'js-test-destination', (response, alertClass) => {
+bindTestForm('test', 'js-test-destination', 'js-destination-form', (response, alertClass) => {
```

Comment:
Maybe could add -button to the end of the `js-test-destination` id

Reply:
all done :)